pom.xml

```diff
@@ -48,15 +48,16 @@
     <parent>
         <artifactId>jahia-modules</artifactId>
         <groupId>org.jahia.modules</groupId>
-        <version>7.1.2.2</version>
+        <version>7.5.0.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>bootstrap3-modules</artifactId>
     <name>Bootstrap3 Modules</name>
     <packaging>pom</packaging>
-    <version>3.7.0-SNAPSHOT</version>
+    <version>4.0.0-SNAPSHOT</version>
     <properties>
         <jahia.package.version>1.0</jahia.package.version>
+        <jahia-module-signature>MCwCFFuYQJ8/rGH8aOS2Tw2tKOBmTGbhAhR07fDq3V5rsXzIfFS9FVHPzSyhng==</jahia-module-signature>
```

Comment:
this is not a module, it's just a package and it's never deployed on the platform, so I don't think you need it here.
To be confirmed by @tdraier

Reply:
I wasn't sure also

Reply:
yes, not needed in pom packaging, only for modules